<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroundRadiationMonitoringDockWidgetBase</class>
 <widget class="QDockWidget" name="GroundRadiationMonitoringDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ground Radiation Monitoring</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="raster_label">
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Raster:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="track_label">
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Track:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="load_raster">
        <property name="text">
         <string>Load raster</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="load_track">
        <property name="text">
         <string>Load track</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsMapLayerComboBox" name="raster_box"/>
      </item>
      <item row="2" column="1">
       <widget class="QgsMapLayerComboBox" name="track_box"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
